packages/common/contact.ts

```diff
@@ -106,9 +107,11 @@ export const toContactMethod = (contact: ContactData | null): ContactMethod | nu
 
   const { stateMethod, emails, faxes } = method
 
+  const formattedFaxes = faxes && faxes.length !== 0 ? faxes.map(item => formatPhoneNumber(item)) : []
```

Comment:
You cannot do it here.  `toContactMethod` is called by `trpc.ts` to get the fax number supplied to Twillio.  Remove this.